vtex-apps/checkout-resources
# Setup checkout-resources with existing queries and mutations

_This PR must be merged **after** https://github.com/vtex/builder-hub/pull/701._

#### What problem is this solving?

This PR creates a new app that will centralize all Checkout IO GraphQL operations. We're doing this in order to make use of reusable [`Fragments`](https://graphql.org/learn/queries/#fragments), similar to what [`store-resources`](https://github.com/vtex-apps/store-resources) already does.

#### How should this be manually tested?

Link this app, then change an existing app to use a query from `vtex.checkout-resources` and verify the query works.

#### Checklist/Reminders

- [x] [Linked this PR to a Clubhouse story (if applicable).](https://app.clubhouse.io/vtex/story/19327/unificar-retorno-das-queries-mutations-graphql)

#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
